package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "@mendix/nanoflow-commons",
   "moduleName": "Nanoflow Commons",
-  "version": "2.6.1",
+  "version": "4.0.3",
```

Comment:
Why did we need to jump from`2.6.1`to `4.0.3`?

Reply:
Fixed